Ktisis/Interface/Components/Transforms/TransformTable.cs

```diff
@@ -115,7 +119,7 @@ public bool DrawPosition(ref Vector3 position, TransformTableFlags flags = Trans
 
 	private bool DrawPosition(ref Vector3 pos, bool op) {
 		var result = this.DrawLinear("##TransformTable_Pos", ref pos);
-		if (op) this.DrawOperation(PositionOp, FontAwesomeIcon.LocationArrow, "transform.position");
+		if (op) this.DrawOperation(PositionOp, FontAwesomeIcon.LocationArrow, this._locale.Translate("transform.position"));
```

Comment:
instead of passing in `this._locale.Translate(...)` all the time - is it worth refactoring this closer to the `DrawOperation`?